src/stacktraces.cpp

```diff
@@ -686,32 +686,34 @@ crash_info GetCrashInfoFromException(const std::exception_ptr& e)
     std::string type;
     std::string what;
 
+    auto getExceptionType = [&]() {
```

Comment:
I think I'd prefer
```suggestion
    std::string getExceptionType = [&]() {
```

Reply:
Thats a lambda/callable object assignment not just a `std::string`. If you want to be explicit about the return value it should be something like

```suggestion
    auto getExceptionType = [&]() -> std::string {
```

or
```suggestion
    std::function<std::string()> getExceptionType = [&]() {
```